docs/examples/tutorial/parallelization/manual_work.pyx

```diff
@@ -0,0 +1,18 @@
+from cython.parallel cimport parallel
```

Comment:
I think I need to exclude this file from the Mac/Clang tests (although I'm not sure how)